### ISSUE TITLE
Clock Tower: Rewind (Steam)

### ISSUE THREAD

**Issue by Rot-gut** [farmerbb/RED-Project#183]
After installing Clock Tower: Rewind on Steam, I see in the install folder it has a roms.pak file (32.5 MB). I imagine this is where the roms are kept, but not sure how to unpack.

I imagine this would contain the translated version of the original game as well as the "Rewind" version based on the fan Deluxe hack.

**Comment by Rot-gut** [comment]
> The "demo.sfc" ROM is presumably for the prototype included in the extras menu.
> 
> https://hiddenpalace.org/Clock_Tower_(Prototype)

Yep, it's accessible in-game via the Extras feature.

I guess without knowing the password users are SOL?

**Comment by Infinest** [comment]
@Rot-gut 
![title](https://github.com/user-attachments/assets/d4d37006-608d-4730-bce2-6994be05ba92)
The password for the zip-archive included as roms.pak is "smns/rbkjcs.rolr"

**Comment by Infinest** [comment]
By the way, do not post this information on the Steam discussions.
It really seems the devs do not want this information out there as they deleted my comment and permanently banned me from their board. 🤔

**Comment by kjetil-f** [comment]
Since this is a Carbon Engine game from LRG, I find it a bit ironic that Modern Vintage Gamer (who works on this stuff) just released a video called "It's up to us to save Video Games".

**Comment by Infinest** [comment]
> Since this is a Carbon Engine game from LRG, I find it a bit ironic that Modern Vintage Gamer (who works on this stuff) just released a video called "It's up to us to save Video Games".

I agree. This is the first time i have the devs noticably go against me for posting such information.
I even posted guides for extracting ROMs on Steam for other releases before and the devs of those games did not care enough to remove them...

**Comment by Rot-gut** [comment]
> Since this is a Carbon Engine game from LRG, I find it a bit ironic that Modern Vintage Gamer (who works on this stuff) just released a video called "It's up to us to save Video Games".

I imagine it's not up to him. If anything, Sunsoft or Capcom (or WayForward) may have mandated that the ROMs be protected.

**Comment by Mysterypome** [comment]
hello I was able to extract the clock tower game and it's various regional versions including the demo version and now I want to mod the graphics to it. Could someone point me to a graphical editor that would help for this game and one that would actually help me the SNES tiles more clearly? Thanks in advance.
